assets/js/modules/adsense/common/site-steps.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * AdSense Site Steps component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import Link from '../../../components/link';
+import ProgressBar from '../../../components/progress-bar';
+import {
+	getAccountSiteURL,
+	getAccountSiteAdsPreviewURL,
+} from '../util/url';
+import { STORE_NAME } from '../datastore';
+import { STORE_NAME as siteStoreName } from '../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as userStoreName } from '../../../googlesitekit/datastore/user/constants';
+const { useSelect } = Data;
+
+export default function SiteSteps() {
+	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
+	const siteURL = useSelect( ( select ) => select( siteStoreName ).getReferenceSiteURL() );
+	const userEmail = useSelect( ( select ) => select( userStoreName ).getEmail() );
+
+	if ( ! accountID || ! siteURL || ! userEmail ) {
```

Comment:
You're checking for false-y values here, but shouldn't these be checks for `undefined`? If `accountID` is `0`, this would fail, no?

Reply:
Not a big deal, I did the false-y check here intentionally, since `accountID` of `0` is not a valid account ID anyway, it's always a non-empty string.

Reply:
Oh, right, cool. 👍